src/containers/Login/style.module.css

```
@@ -0,0 +1,46 @@
+.container {
+  width: 100vw;
+  height: 100vh;
+
+  background-color: white;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.wrapper {
+  width: 350px;
+  height: 250px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  position: relative;
```

remover espaço em branco
```suggestion
  display: flex;
  flex-direction: column;
  align-items: center;
  position: relative;
```